packaging/packaging-practices/en.md

```diff
@@ -196,6 +196,7 @@ In output:
 Provides: pkgconfig(gtk+-3.0) pkgconfig(gdk-3.0) pkgconfig(gdk-wayland-3.0) pkgconfig(gail-3.0) pkgconfig(gdk-x11-3.0) pkgconfig(gtk+-unix-print-3.0)
 pkgconfig(gtk+-wayland-3.0) pkgconfig(gtk+-x11-3.0) 
 ```
```

Comment:
Should be a newline between the paragraph and end of the code block.

Reply:
I hope this is what you meant

Reply:
No the newline should be between the paragraph and the **end** of the code block (so the ```)

Reply:
Sorry, I don't think I get it. Now is that it?

Reply:
I meant between the pkgconfigs code block and the paragraph, but I'd keep the other newline you just added too :D

Reply:
Oh, sorry, I thought about different code block